code/controllers/subsystems/trade.dm

```diff
@@ -446,7 +446,7 @@ SUBSYSTEM_DEF(trade)
 	var/obj/structure/closet/secure_closet/personal/trade/C
 	var/count_of_all = collect_counts_from(shopList)
 	var/price_for_all = collect_price_for_list(shopList)
-	if(isnum(count_of_all) && count_of_all > 1)
+	if((isnum(count_of_all) && count_of_all > 1) || shopList.Find("/obj/machinery/power/supermatter") > -1)
```

Comment:
From byond manual

> The Find proc could be used for that purpose, but if you don't care about the position of the item in the list, in is more convenient and efficient. The expression is 1 if the item exists in the list and 0 if not. 

Your `> -1` will always be true ?

Reply:
I didn't realize, I just thought it would return the position and -1 if it doesn't exist like a normal language. I'll fix this

Reply:
change has been applied, searches for >0 rather than >-1

Reply:
> like a normal language

This is BYOND we're dealing with my good sir!